saleor/core/utils/anonymization.py

```diff
@@ -0,0 +1,71 @@
+import copy
+from typing import TYPE_CHECKING
+
+from faker import Faker
+
+from ...account.models import Address, User
+
+if TYPE_CHECKING:
+    from ...order.models import Order
+    from ...checkout.models import Checkout
+
+fake = Faker()
+
+
+def generate_fake_address() -> "Address":
```

Comment:
In `saleor/core/utils/random_data.py` we have a bunch of functions that do the same functionality:
- `create_address` which also handles generating valid states and postal codes for the US addresses
- `create_fake_user` 

Since we have a module for generating random data, I think we should keep all related functions there and try to reuse ones that do the same thing.

Reply:
I've reused the `create_address` and `create_fake_user` from the `random_data`. Though I had to add the change to make the `saving` optional as we don't want to save new or update the existing instances.

I left the functions in the `anonymization` module. Its purpose is different from the `random_data`. The `anonymization` doesn't want to save/update anything in the database. Although, because of the complex structure of relations it wants to be based on an already existing instances.